ScheduleAPITests/Models/FakeDateNowRetriever.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using ScheduleAPI.Models;
+
+namespace ScheduleAPITests.Models
+{
+    public class FakeDateNowRetriever : IDateNowRetriever
+    {
+        public DateTime UtcNow { get; }
+
+        public FakeDateNowRetriever(DateTime utcNow)
```

Comment:
Should we combine this logic into 1 actual class, the DateTimeNow class via constructors?
i.e. parameterless constructor sets internal DateTime to null, parameterized constructor sets internal DateTime, then properties handle the null case?

Reply:
I personally lean towards having test implementations if only to prevent misuse of the actual API. Keeps the intentions behind the API uniform to intent of use. 
We can discuss.

Reply:
yeah, that does add some decent protection against misuse... fair enough, I'd say leave it be then! +1